inc/template-tags.php

```diff
@@ -158,47 +158,36 @@ function primer_paging_nav() {
 /**
  * Display navigation to next/previous post when applicable.
  *
- * @global WP_Post $post
+ * @uses the_post_navigation
+ * @uses primer_post_nav_text
+ *
  * @since  1.0.0
  */
 function primer_post_nav() {
```

Comment:
@EvanHerman We could expose an `$args` param here which would be applied to `the_post_navigation( $args )` after applying defaults with `wp_parse_args`.